manifests/ovs-net-vlan100.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
IMO network definition should not be a part of kubevirtci, it should be created as a part of testing.

Reply:
Yes we don't need this anymore. We create them on the functest

https://github.com/kubevirt/kubevirt/pull/1583/files#diff-897951be21489aebaa1df0c66894562cR60